run-local.sh

```diff
@@ -65,13 +69,33 @@ export SPRING_DATASOURCE_URL='jdbc:postgresql://${DB_SERVER}/${DB_NAME}'
 export POLICYV3_ENABLED=true
 export RECALL_ENABLED=true
 
-if [[ $1 != "--skip-docker" ]]; then
+SKIP_DOCKER=false
+DEBUG=""
+
+# Parse all arguments
+for arg in "$@"; do
+  case $arg in
+    --skip-docker)
+      SKIP_DOCKER=true
+      ;;
+    --debug)
+      DEBUG="--debug-jvm"
```

Comment:
this requires you to connect the debugger and blocks the app starting up until you do